packages/commerce-sdk-react/src/hooks/index.ts

```diff
@@ -15,6 +15,9 @@ export * from './ShopperProducts'
 export * from './ShopperPromotions'
 export * from './ShopperSearch'
 export * from './useAuthHelper'
+export {default as useAccessToken} from './useAccessToken'
```

Comment:
```suggestion
export useAccessToken from './useAccessToken'
```
[NIT] That's the magic of `default` 😜

Reply:
my IDE hates this, is there another way to do this as a one liner? 
<img width="682" alt="Screenshot 2023-03-13 at 10 51 10 AM" src="https://user-images.githubusercontent.com/10948652/224786382-b69d4022-8fdc-499f-84ca-a91413fd3cc0.png">

Reply:
That's weird. Does your IDE complain when you open other files (like `useAuthContext.ts`)? Seems like they use that syntax without any trouble.